backend/app/services/device/cloud_provider.py

```
@@ -0,0 +1,162 @@
+        device_id: str,
+    ) -> Optional[Dict[str, Any]]:
+        """Get cloud device status from Redis and database."""
+        device_kind = (
+            db.query(Kind)
+            .filter(
+                and_(
+                    Kind.user_id == user_id,
+                    Kind.kind == "Device",
+                    Kind.namespace == "default",
+                    Kind.name == device_id,
+                    Kind.is_active == True,
```

Ruff (0.15.4)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)